cmake/Install.cmake

```diff
@@ -629,7 +629,7 @@ if(WIN32 AND NOT UNIX)
 
   # ConvertInputFormat is added via add_subdirectory, and in there we set CMAKE_INSTALL_OPENMP_LIBRARIES at parent scope already
   # Otherwise, if kiva (no by default) **actually** linked to OpenMP, we need to ship the libs
-  if (NOT ${CMAKE_INSTALL_OPENMP_LIBRARIES} AND (${ENABLE_OPENMP}))
+  if ((NOT CMAKE_INSTALL_OPENMP_LIBRARIES) AND ENABLE_OPENMP)
```

Comment:
> The if() command has a special condition syntax that allows for variable references in the short form <variable> instead of ${<variable>}. However, environment variables always need to be referenced as $ENV{<variable>}.

```
CMake Error at cmake/Install.cmake:632 (if):
  if given arguments:

    "NOT" "AND" "(" "OFF" ")"

  Unknown arguments specified
Call Stack (most recent call first):
  CMakeLists.txt:430 (include)
```